### ISSUE TITLE
footnotes inside .column-margin

### ISSUE THREAD

**Issue by aborruso** [quarto-dev/quarto-cli#5338]
### Bug description

Hi
as I have written [here](https://github.com/quarto-dev/quarto-cli/issues/5289#issuecomment-1526507104), using quarto 1.4.11 and running `quarto render tmp-lite.qmd` ([tmp-lite.zip](https://github.com/quarto-dev/quarto-cli/files/11348285/tmp-lite.zip)) the footnotes are inside `.column-margin`.

Am I doing something wrong?

My OS is Debian 11.

![image](https://user-images.githubusercontent.com/30607/234999618-e3b5a36f-4d69-4696-9be4-db73ab356200.png)

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.
- [X] Please provide the output of `quarto check` so we know which version of quarto and its dependencies you're running.

**Comment by aborruso** [comment]
@dragonstyle I don't want to pressure you, just to know if you've verified that the bug exists. 
I'm trying to fix it myself, but I don't have the skills to do it.

Thanks again for what you are doing to make this amazing product

**Comment by aborruso** [comment]
> I'll try to take a look at it later this week, though it may prove pretty challenging so I'm not sure I'll get to it quickly.

Thank you very much.

I have seen that it's in this way also the [quarto website](https://quarto.org/docs/authoring/footnotes-and-citations.html#footnotes)

![image](https://user-images.githubusercontent.com/30607/236218563-be90a7df-5c34-41f5-8e9b-c3a7fa3d7a6c.png)

**Comment by dragonstyle** [comment]
Thank you for your persistence on this issue- this is definitely a bad regression and was different than I was anticipating!

**Comment by aborruso** [comment]
> Thank you for your persistence on this issue- this is definitely a bad regression and was different than I was anticipating!

I have installed 1.4.36, and I have the same behavior: footnotes inside .column-margin, and not at the end.

And I have set `reference-location: document`.

![image](https://user-images.githubusercontent.com/30607/236327955-a176a3bf-26ad-4054-be7d-c55ef6ebdf13.png)

**Comment by dragonstyle** [comment]
Apologies - that was an incomplete fix! I've made a second attempt and a fresh build is cooking now. It should be available in 15-20 minutes...

**Comment by aborruso** [comment]
> Apologies - that was an incomplete fix! I've made a second attempt and a fresh build is cooking now. It should be available in 15-20 minutes...

wow, thank you, it works great!!

@dragonstyle do you plan to correct #5289 also for the 1.3 release? Because if I understand correctly, now it is only active for 1.4.

Thank you again!!!

**Comment by dragonstyle** [comment]
I this does appear to only affect 1.4 so we aren't planning to back port this to 1.3!

**Comment by aborruso** [comment]
> I this does appear to only affect 1.4 so we aren't planning to back port this to 1.3!

Hi @dragonstyle I'm referencing to #5289, and to column-margin wrong positioned. Exploring that issue I have discovered this one related to footnotes.

If you use the [temp file](https://github.com/quarto-dev/quarto-cli/files/11348285/tmp-lite.zip) I have inserted at start, and you run `quarto render tmp-lite.qmd` using `1.3`, you have the column-margin wrong positioned.

It seems to me important to solve it also in stable version. Did you solve this also in 1.3?

Thank you

![image](https://user-images.githubusercontent.com/30607/236692511-1857bf81-53d4-4722-b9ee-029429c4a55c.png)
